src/redux/selectors/index.js

```diff
@@ -58,45 +57,13 @@ export const activeCollectionShowMarkedOnlySelector = createSelector(
 );
 
 export const patientSelector = createSelector(
-  [resourcesSelector, resourceIdsGroupedByTypeSelector],
-  (resources, resourceIdsGroupedByType) => {
-    const patient = resourceIdsGroupedByType?.Patient;
-    if (!patient) {
-      return null;
-    }
-    const patientId = Array.from(patient?.Other)[0];
-    return resources[patientId];
-  },
+  [resourcesSelector],
+  (resources) => values(resources).find((r) => r.type === 'Patient'),
 );
 
 export const providersSelector = createSelector(
-  [resourcesSelector, resourceIdsGroupedByTypeSelector],
-  (resources, resourceIdsGroupedByType) => {
-    const serviceProviders = resourceIdsGroupedByType?.Organization?.Other;
-    if (serviceProviders) {
-      return Array.from(serviceProviders).map((id) => resources[id]);
-    }
-    return [];
-  },
-);
-
-// eslint-disable-next-line max-len
-const sortEntriesByResourceType = ([t1], [t2]) => ((PLURAL_RESOURCE_TYPES[t1].toLowerCase() < PLURAL_RESOURCE_TYPES[t2].toLowerCase()) ? -1 : 1);
-
-export const supportedResourcesSelector = createSelector(
-  [resourceIdsGroupedByTypeSelector],
-  (resourceIdsGroupedByType) => Object.entries(resourceIdsGroupedByType)
-    // do not include Patient, Observation, or unknown/unsupported:
-    .filter(([type]) => !!PLURAL_RESOURCE_TYPES[type]) // derived "type" includes: lab/vitals
-    .sort(sortEntriesByResourceType)
-    .reduce((acc, [resourceType, subTypes]) => {
-      const totalCount = values(subTypes).reduce((count, idSet) => count + idSet.size, 0);
-      return acc.concat({
-        resourceType,
-        totalCount,
-        subTypes,
-      });
-    }, []),
+  [resourcesSelector],
+  (resources) => values(resources).filter((r) => r.type === 'Organization'),
```

Comment:
this is possibly naive -- treating every organization as a "provider"

However, this is what the implementation in `origin/master` already does.

...I'll attempt to fix in a follow up PR, which may need to map the "reference" type to a set of resource IDs.  This may also be applicable to "Practitioner"